WordPress/Classes/ViewRelated/Me/App Settings/AppSettingsViewController.swift

```diff
@@ -220,6 +253,33 @@ class AppSettingsViewController: UITableViewController {
         }
     }
 
+    func imageOptimizationChanged() -> (Bool) -> Void {
+        return { [weak self] value in
+            MediaSettings().imageOptimizationSetting = value
+
+            // Track setting changes
+            WPAnalytics.track(.appSettingsOptimizeImagesChanged, properties: ["enabled": value as
+                                                                                     AnyObject])
```

Comment:
Can we drop the cast here?

```suggestion
            WPAnalytics.track(.appSettingsOptimizeImagesChanged, properties: ["enabled": value])
```

Reply:
Sure! I used the casting here following other calls to analytics we have along the file ([example](https://github.com/wordpress-mobile/WordPress-iOS/blob/a51fb8fea237faf4b0691484fc3d84d99a787c62/WordPress/Classes/ViewRelated/Me/App%20Settings/AppSettingsViewController.swift#L252)). But I agree that seems unnecessary based on the `properties` type. I'll remove the casting from others as well 👍.

Reply:
Suggestion applied in https://github.com/wordpress-mobile/WordPress-iOS/pull/21981/commits/f29f4d2837d4a1d17b2d4f182f178680d8df74b6.